public/assets/js/scripts.js

```diff
@@ -285,4 +286,49 @@ $(document).ready(function () {
     $(this).html() == '<span><img src="./assets/img/icons/close-map-pin.svg" alt="view map"> &nbsp; Close Map</span>' ? $(this).html('<span><img src="./assets/img/icons/view-map-pin.svg" alt="view map"> &nbsp; View Map</span>') : $(this).html('<span><img src="./assets/img/icons/close-map-pin.svg" alt="view map"> &nbsp; Close Map</span>');
   });
 
+  $('.search-calculate-button').click(function(){
+    const periodic = $('.periodic-investment').val();
+    const initial = $('.initial-investment').val();
+    const avgPropertyCost = removeCommasAndMakeNumber($('.investment-value-input').data('max-amount'));
+    const frequency = $("input[name='investment-frequency']:checked").val();
+
+    let balance = avgPropertyCost - initial;
+    let output;
+    let paymentType;
+    
+    let outrightPersonal = initial >= (avgPropertyCost * 0.5) && balance / periodic < 6 / frequency;
+    let immediatePrivate = initial >= (avgPropertyCost * 0.25) && (avgPropertyCost <= 45000000);
+    let outrightMortgage = initial >= (avgPropertyCost * 0.25);
+    let directSpread = initial >= (avgPropertyCost * 0.2) && balance / periodic > 6 / frequency && balance / periodic <= 24 / frequency;
+    let immediateFederal = initial >= (avgPropertyCost * 0.1) && (avgPropertyCost <= 15000000) && balance / periodic <= 12 / frequency;
+    let spreadFederal = initial >= (avgPropertyCost * 0.01) && (avgPropertyCost <= 15000000) && balance / periodic <= 24 / frequency;
+    let rentToOwn = initial >= (avgPropertyCost * 0.05) && balance / periodic <= 120 / frequency;
+    let spreadPrivate = initial >= (avgPropertyCost * 0.025) && (avgPropertyCost <= 45000000) && balance / periodic <= 24 / frequency;
+    
+
+    if (outrightPersonal) { // 50%
+      paymentType = 'Personal Outright Payment';
+    } else if (immediatePrivate) { // 25% < 45m
+      paymentType = 'Private Immediate Equity';
+    } else if (outrightMortgage) { // 25%
+      paymentType = 'Mortgage Outright Payment';
+    } else if (directSpread) { // 20%
+      paymentType = 'Direct Spread Payment';
+    } else if (immediateFederal) { // 10% < 15m
+      paymentType = 'Federal Immediate Equity';
+    } else if (spreadFederal) { // 1% < 15m
+      paymentType = 'Federal Spread Equity';
+    } else if (spreadPrivate) { // 2.5% < 45m
+      paymentType = 'Private Spread Equity';
+    } else if (rentToOwn) { // 5%
+      paymentType = 'Rent to own package';
+    } else { // 1%
+      paymentType = 'Hybrid Outright Payment';
+    }
+
+    output = `${paymentType} <img src="./assets/img/icons/question-mark.svg" alt="payment">`
```

Comment:
We might want to display more than one output at time. Can we update the output to cater for multiple categories?